kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkManager.java

```diff
@@ -59,6 +63,9 @@ private static ExecutorService queryThreadPool() {
     SearchResult<T> errorResult =
         new SearchResult<>(new ArrayList<>(), 0, 0, new ArrayList<>(), 0, 0, 1, 0);
 
+    // https://stackoverflow.com/questions/69647913/how-to-propagate-brave-trace-in-lettuce-client-async-command
```

Comment:
Since, this is not a bug, instead of a stack overflow link can you please add a link to brave documentation here if possible.